ingestion_tools/scripts/schema_migration/upgrade.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+from collections import OrderedDict
+from typing import Any
+
+import yaml
+
+from schema_migration import migrate_v1_0_0, migrate_v1_1_0
+
+logger = logging.getLogger(__name__)
+
+
+def upgrade_config(data: dict[str, Any]) -> dict[str, Any]:
+    version_map = OrderedDict(
+        # Version_map must be updated when a new migration is needed.
+        # Order matters.
+        {
+            # current_version: (update_function, next_version)
+            "0.0.0": (migrate_v1_0_0.upgrade, "1.0.0"),
+            "1.0.0": (migrate_v1_1_0.upgrade, "1.1.0")})
```

Comment:
NIT: it'd be nice to get to the point where the script is smart enough to read the list of migrations from a directory and check the version tags in each config file against the list of migrations available, and then runs only the necessary migrations (more like the way alembic works) but this is a good start!